src/components/StatsGroup/statsGroup.stories.ts

```diff
@@ -0,0 +1,133 @@
+                fill="#FF462D"
+                mask="url(#path-1-outside-1_3819_64794)"
+              />
+            </svg>
+          </div>
+          <p class="kd-type--body-02 kd-type--weight-regular">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Arcu
+            facilisi pellentesque morbi molestie sed ut at sed ac. Semper tortor
+            feugiat non sem eget libero. Risus semper facilisis convallis vitae.
+          </p>
+        </div>
+
+        <div slot="text-three">
+          <div class="title">
+            <svg
```

Comment:
We need to check with design. These probably shouldn't be svgs. SVGs lack accessibility and number formats change based on nation.